test/general/TestIDs.py

```diff
@@ -18,18 +18,18 @@ def testUniqueLocations(self):
             self.assertEqual(len(known_location_ids) - len(world_type.location_id_to_name), current)
 
     def testRangeItems(self):
-        """There are Javascript clients, which are limited to 2**53 integer size."""
+        """There are Javascript clients, which are limited to +/- 2**53 integers due to 64bit float precision."""
         for gamename, world_type in AutoWorldRegister.world_types.items():
             with self.subTest(game=gamename):
                 for item_id in world_type.item_id_to_name:
-                    self.assertLess(item_id, 2**53)
+                    self.assertLessEqual(item_id, 2**53)
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER the limit iss 2**53-1, so less than should apply, not less or equal.

Reply:
That's interesting because
```python
>>> 4*(2**51) == 2**53
True
```

Reply:
I mean, killing off *one* ID to be on the safe side is probably fine.

Reply:
it's more about the annoying `-1` everywhere :dagger: 

Will rework the PR if you want (probably force-pushing since that commit history makes no sense then)

Reply:
Just say "less than", gets rid of the -1 :P

Reply:
sadly "less than" does not work for +/-. Anyway, PR updated